source-code/server.lua

```diff
@@ -89,7 +94,17 @@ httpd_set("/status", function(request, response)
 end)
 
 httpd_set("/restart", function(request, response)
-  tmr.create():alarm(5000, tmr.ALARM_SINGLE, node.restart)
+  print("Heap: ", node.heap(), "HTTP: ", "Restart")
+  tmr.create():alarm(5000, tmr.ALARM_SINGLE, function() node.restart() end)
```

Comment:
Instead of creating a new function, I think you can just pass the function `node.restart`. Any reason you changed that?